tests/netpyne-tests.js

```diff
@@ -21,7 +23,11 @@ casper.test.begin('NetPyNE projects tests', function suite(test) {
   casper.options.waitTimeout = 10000
   casper.on("page.error", function(msg, trace) {
     this.echo("Error: " + msg, "ERROR");
-  });
+	});
+	
+	// casper.on('remote.message', function(message) { // UNCOMMENT TO GET MORE DEBUGGING INFO THAN WHAT YOU PROBABLY WANT
```

Comment:
what is this? what extra info do we get here?

Reply:
you capture the console.log() there. so if the console gives you an error because netpyne tried to import something and failed in the backend, you will see the error.
But this is super verbose, because it also captures other stuff. but it is useful because you can run casper instead of having to try yourself to understand what is failing